tendrl/monitoring_integration/graphite/graphite_utils.py

```diff
@@ -26,6 +26,27 @@ def archive(
         logger.log("debug", NS.get("publisher_id", None),
                    {'message': msg})
 
+def create_cluster_alias(cluster_details):
+    for cluster in cluster_details:
+        integration_id = cluster.get("integration_id")
+        short_name = cluster.get("short_name")
+        alias_dir_path = "%sname" % get_data_dir_path()
+        if not os.path.exists(alias_dir_path):
+            try:
+                os.makedirs(str(alias_dir_path))
+            except OSError as ex:
+                logger.log("debug", NS.get("publisher_id", None),
+                   {'message': str(ex)})
+        if short_name in [None, ""]:
+            short_name = integration_id
+        try:
+            os.symlink(
+                "%s/clusters/%s" % (get_data_dir_path(), integration_id),
```

Comment:
Are we trying to create the symlink during every sync?

Reply:
ack